pep-0597.rst

```diff
@@ -33,135 +34,140 @@ Using the default encoding is a common mistake
 Developers using macOS or Linux may forget that the default encoding
 is not always UTF-8.
 
-For example, ``long_description = open("README.md").read()`` in
-``setup.py`` is a common mistake. Many Windows users can not install
-the package if there is at least one non-ASCII character (e.g. emoji)
-in the ``README.md`` file which is encoded in UTF-8.
+For example, using ``long_description = open("README.md").read()`` in
+``setup.py`` is a common mistake. Many Windows users cannot install
+such packages if there is at least one non-ASCII character (e.g. emoji)
+in their UTF-8-encoded ``README.md`` file.
```

Comment:
Here you previously mentioned loading "TOML", but I was very confused what you were referring to as it didn't connect to the previous sentance and the only commonly-used TOML file I'm aware of in packaging is `pyproject.toml`, which AFAIK is not loaded programmatically in setup.py. Is there something I'm missing here?